### DISCUSSION TITLE
Problem with downloaded mumps using update_and_rebuild_petsc.sh

### DISCUSSION THREAD

**Discussion by TheBEllis** [idaholab/moose#21335] [Q&A Getting Started] [answered]
I'm currently trying to build MOOSE in an ubuntu WSL environment, and am running into issues with the petsc build script. The error I get at the moment is this. I can build MOOSE fine in my actual Linux install, just wsl that is giving me issues at the moment.

`Installing Mumps; this may take several minutes`
`TESTING: check from config.libraries(config/BuildSystem/config/libraries.py:157)`
`UNABLE to CONFIGURE with GIVEN OPTIONS    (see configure.log for details):`
`Downloaded mumps could not be used. Please check install in /home/bill/projects/moose/petsc/arch-moose`

I have uploaded the configure.log here.
[configlog.txt](https://github.com/idaholab/moose/files/8926862/configlog.txt)

Any help greatly appreciated :)

**Comment by GiudGiud** [comment, marked as answer]
Hello

In a WSL environment you do not really want to use the manual installation of petsc and libmesh, it's much easier to use the conda build:
https://mooseframework.inl.gov/getting_started/installation/conda.html
btw did you look at the initial instructions for wsl:
https://mooseframework.inl.gov/getting_started/installation/windows10.html
Guillaume

**Reply by TheBEllis** [reply]
Apologies I did not see the windows instructions, I just went to use the linux ones out of habit! Thank you so much, I will give them a go!
